sdks/python/apache_beam/typehints/trivial_inference_test_py3.py

```diff
@@ -43,8 +42,8 @@ def testBuildListUnpack(self):
 
   def testBuildTupleUnpack(self):
     # Lambda uses BUILD_TUPLE_UNPACK opcode in Python 3.
-    self.assertReturnType(typehints.Tuple[int, str, str],
-                          lambda _list1, _list2: (*_list1, *_list2, *_list2),
+    self.assertReturnType(typehints.Tuple[int, str, str], lambda _list1, _list2:
+                          (*_list1, *_list2, *_list2),
```

Comment:
ALLOW_MULTILINE_LAMBDAS = false

Reply:
Weird. Looks like this knob is turned off by default, but the lambda is getting split anyway

Reply:
I've disabled formatting for this block